mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+PhysicalSizeXUnit,The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um),,,FALSE,,,,HTAN,
+PhysicalSizeY,Physical size (Y-dimension) of a pixel. Units are set by PhysicalSizeYUnit. Floating point value > 0.,,,FALSE,,,,HTAN,
+PhysicalSizeYUnit,The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um),,,FALSE,,,,HTAN,
+PhysicalSizeZ,Physical size (Z-dimension) of a pixel. Units are set by PhysicalSizeZUnit. Floating point value > 0.,,,FALSE,,,,HTAN,
+PhysicalSizeZUnit,The units of the physical size of a pixel. See OME enumeration of allowed values for the UnitsLength attribute -- default: microns (um),,,FALSE,,,,HTAN,
+Pixels BigEndian,Boolean (True/False),,,FALSE,,,,HTAN,
+PlaneCount,"Number of Z-planes (not to be confused with downsampled ""pyramid""). Integer >=1",,,FALSE,,,,HTAN,
+SizeC,Number of channels. Integer >= 1,,,FALSE,,,,HTAN,
+SizeT,Number of time points. Integer >= 1,,,FALSE,,,,HTAN,
+SizeX,Size of image: X dimension (in pixels). Integer >= 1,,,FALSE,,,,HTAN,
+SizeY,Size of image: Y dimension (in pixels). Integer >= 1,,,FALSE,,,,HTAN,
+SizeZ,Size of image: Z dimension (in pixels). Integer >= 1,,,FALSE,,,,HTAN,
+PixelType,"Data type for each pixel value. E.g. ""uint16""",,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Spatial Transcriptomics Level 3,Files contain processed data from the NanoString GeoMx DSP Pipeline.,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of Parent Data File, Synapse ID GeoMx ROI Segment Annotation File, GeoMx DSP Unique Probe Count, GeoMx DSP Unique Target Count, Matrix Type, GeoMx DSP Genomic Reference, GeoMx DSP Workflow Type, GeoMx DSP Workflow Parameter Description, GeoMx DSP Workflow Link",FALSE,,,,HTAN,
+Synapse ID of Parent Data File,Synapse ID indicating the file(s) from which these files were derived; multiple parent files should be comma-separated,,,TRUE,,,,HTAN,
+GeoMx DSP Unique Probe Count,Total number of unique probes reported.,,,FALSE,,,,HTAN,
+GeoMx DSP Unique Target Count,Total number of unique genes reported.,,,FALSE,,,,HTAN,
+Matrix Type,Type of data stored in matrix.,,,FALSE,,,,HTAN,
+GeoMX ROI Segment Annotation,"GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI and AOI coordinates. ",,"Component, Parent Biospecimen ID, Scan name, ROI Name, AOI Name, Tags, ROI X Coordinate, ROI Y Coordinate, AOI X Coordinate, AOI Y Coordinate, QC status, Scan Height, Scan Width, Scan Offset X, Scan Offset Y, Binding Density, Positive norm factor, Surface area, Nuclei count, Tissue Stain, Slide name, Raw reads, Stitched reads, Aligned reads, Deduplicated reads, In Situ Negative median, Biological probe median, Sequencing Saturation, MapQ30",FALSE,,,,HTAN,
```

Comment:
This is a combination of features from the RCC and DCC versions of the ROI annotation file defined in the HTAN model. We can define dependent attributes to indicate which ones are required for each file format, but I wanted to make sure we have the features available.

Since there can be multiple AOIs within an ROI, I added coordinates for AOI regions as well. This can be removed if unnecessary.